### PR TITLE
SOC-1043 Use https url for tracking events internally

### PR DESCRIPTION
**Pull request by bkoval**
@lizlux @rogatty @mklucsarits @garthwebb @rafalkalinski 
https://wikia-inc.atlassian.net/browse/SOC-1043
We're changing tracker URL as we discovered http:// call is getting blocked on https pages.

It follows https://github.com/Wikia/wikia-vcl/pull/526

### CODE REVIEW

**Comment by mklucsarits** [comment]
:+1:

**Comment by lizlux** [comment]
LGTM

**Comment by garthwebb** [comment]
LGTM

File: front/scripts/mercury/modules/Trackers/Internal.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ interface InternalTrackingParams extends TrackingParams {
 
 module Mercury.Modules.Trackers {
 	export class Internal {
-		baseUrl: string = 'http://a.wikia-beacon.com/__track/';
+		baseUrl: string = 'https://beacon.wikia-services.com/__track/';
```
Comment (rogatty) [inline front/scripts/mercury/modules/Trackers/Internal.ts]: Would `//beacon.wikia-services.com/__track/` work here?

**Comment by lizlux** [comment]
I'm inclined to follow Paul Irish's idea that protocol relative URLs for assets are an anti-pattern http://www.paulirish.com/2010/the-protocol-relative-url/. Supposedly HTTPS is just as fast as HTTP and it's more secure, and it's the way the web is moving. 

Also found this: https://developers.google.com/web/fundamentals/discovery-and-distribution/security-with-https/make-intra-site-urls-relative?hl=en
>NOTE: It is perfectly OK to include HTTPS resources in an HTTP page.

**Comment by rogatty** [comment]
@lizlux: if https:// works without problems in Mercury then I have nothing against using it.

**Comment by bkoval** [comment]
Sorry guys, I merged it here to test around on sandbox. I'll bring back the https :)

**Comment by bkoval** [comment]
It was tested in the original form (with https) by our QM, including regression tests for Mercury tracking.

**Comment by lizlux** [comment]
Okay cool, yeah, glad it went though some good QM. Thanks!

**Pull request merged**